src/utils/jsonUtils.ts

```diff
@@ -7,6 +7,7 @@ function convertDateHelper(key: any, value: any): any {
         if (a) {
             return new Date(value);
         }
+        r;
```

Comment:
slette?

Reply:
La den til nå for å teste at den checker ut rett branch. Vil bare se at builden feiler, så skal jeg fjerne den igjen